static/js/compose.js

```diff
@@ -1006,12 +1006,8 @@ exports.initialize = function () {
             video_call_link = "https://hangouts.google.com/hangouts/_/" + page_params.realm_google_hangouts_domain + "/" + video_call_id;
             insert_video_call_url(video_call_link, target_textarea);
         } else if (page_params.realm_video_chat_provider === "Zoom") {
-            channel.get({
-                url: '/json/calls/create',
-                success: function (response) {
-                    insert_video_call_url(response.zoom_url, target_textarea);
-                },
-            });
+            var redirect_location = window.location.protocol + '//' + window.location.host + '/json/calls/register_zoom_user'
+            window.location.href = redirect_location;
```

Comment:
I assume this is intended to have a conditional for whether they're already registered?